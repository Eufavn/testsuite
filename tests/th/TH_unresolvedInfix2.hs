module TH_unresolvedInfix2 where

import TH_unresolvedInfix_Lib
import Language.Haskell.TH

expr = $( infixE Nothing plus (Just $ n +? n) )
