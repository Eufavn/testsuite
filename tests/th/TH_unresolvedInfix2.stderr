
TH_unresolvedInfix2.hs:6:11:
   The operator `:+' [infixl 6] of a section
       must have lower precedence than that of the operand,
         namely `:+' [infixl 6]
       in the section: `:+ N :+ N'
   In the result of the splice:
     $(infixE Nothing plus (Just $ n +? n))
   To see what the splice expanded to, use -ddump-splices
   In the expression: $(infixE Nothing plus (Just $ n +? n))
   In an equation for `expr':
       expr = $(infixE Nothing plus (Just $ n +? n))
